module-2/backend/week_two.md

```diff
@@ -5,17 +5,45 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. At a high level, what is ActiveRecord? What does it do/allow you to do?
+  A database which can utilize relationships between sets of data.
```

Comment:
ActiveRecord is an ORM. An ActiveRecord class maps to a table in our database. It wraps our queries in Ruby methods so we can easily use them in our code